README.md

```diff
@@ -301,15 +301,33 @@ Asset information
     - `symbol` Asset identification
     - `description` Basic information of assets
     - `suggesteGasLimit` Suggested gas limit. It is like the `gasLimit` field in transaction. The default is 60000
-    - `stop` Whether to stop all actions of the asset. The default is ` false`
+    - `stop` Whether to stop all actions of the asset. The default is `false`. Set it to `true` can stop all the assets other than the query operation
 
 <a name="data-asset-category"></a>
 
 category | description
 --- |---
-1 |  TokenAsset, similar to ethereum's ERC20 token, it can set whether additional issuance is allowed in the future when issuing, and users can divide and transfer money at will after receiving the coin
-2 |  NonFungibleAsset, Similar to ethereum's ERC721 token, it can carry a small amount of asset information and is indivisible
-3 |  CommonAsset, more flexible digital assets, are suitable for more complex scenarios, both of which are specialized from generic assets
+1 |  TokenAsset. Similar to Ethereum's ERC20 token. The issuer can decide whether additional issuance is allowed in the future when issuing. The token is divisible.
+2 |  NonFungibleAsset. Similar to Ethereum's ERC721 token, it can store a certain amount of information. Every token is indivisible.
+3 |  CommonAsset. More flexible digital assets. It is suitable for more complex scenarios. Actualy, the TokenAsset and NonFungibleAsset are two special CommonAsset.
+
+<a name="data-structure-equity"></a>
+
+#### equity
+
+Record the asset information of the asset holder and keep it in the account of the asset holder
+
+```json
+{
+    "assetCode": "0xd0befd3850c574b7f6ad6f7943fe19b212affb90162978adc2193a035ced8884",
+    "assetId": "0x34b04e018488f37f449193af2f24feb3b034c994cde95d30e3181403ac76528a",
+    "balance": "15000000000000000000"
+}
+```
+
+-   `assetCode` Asset code, obtained when the asset is created
+-   `assetId` The assetId, which is obtained when the asset is issued, is the same assetCode as the assetId when the asset type is 1
```

Comment:
-   `assetId` The assetId, which is obtained when the asset is issued. It will be same as assetCode when the asset type is 1